codalab/model/sqlite_model.py

```diff
@@ -0,0 +1,20 @@
+"""
+SQLite is a subclass of BundleModel that stores metadata on a SQLite
```

Comment:
We used to have an SQLite option, but got rid if it because it didn't support all the things we needed (like multirow insert, IIRC)?  Is that going to be a problem here?

Reply:
I think we actually got rid of it because it was just unused code (see #1333). This PR adds the SQLite support back in in a more simplified way than the way it was initially implemented, though (see #1351).

I think we can continue to use SQLite, and if we get to problems with it we can consider modifying it. The advantages of being able to create and manipulate a fully in-memory database before each test case are very good, though.